pylasu/model/model.py

```diff
@@ -180,7 +212,8 @@ def source(self) -> Optional[Source]:
 
     @internal_property
     def properties(self):
-        return (PropertyDescription(p.name, p.provides_nodes, p.multiplicity, getattr(self, p.name))
+        return (PropertyDescription(p.name, p.type, p.is_containment, p.is_reference, p.multiplicity,
```

Comment:
when there are many parameters and several of them are boolean, we could pass them by name to avoid possible issues (i.e., writing them in the wrong order)